src/assets/stylesheets/index.scss

```diff
@@ -356,3 +356,45 @@ body {
   }
 }
 
+#mc_embed_signup {
```

Comment:
nit: I would continue using `:local(.class-names)` where possible. I don't see a reason for this to be an id.